opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -396,6 +341,12 @@ def unregister_observer(self, observer: "Observer") -> None:
         with self.observers_lock:
             self.observers.remove(observer)
 
+    def register_view(self, view):
```

Comment:
this registers view for a particular `Meter` right? I'd say Views are more global thingie, and hence must be part of MeterProvider.
The application may not have access to the Meter itself to add views to it.

Reply:
Are `Views` global? A use case might be defining different views depending on what the meter (or source) is.